### ISSUE TITLE
Add tooltip/alt text with place tip info to search result list

### ISSUE THREAD

**Issue by julietteculver** [pelagios/pelagios-widgets#95]
We have icons for the different place types but currently no alt text, which we need for reasons of accessibility

**Comment by julietteculver** [comment]
Might need to just add the image with an image tag rather than via CSS
